docs/standard/microservices-architecture/microservice-ddd-cqrs-patterns/infrastructure-persistence-layer-design.md

```diff
@@ -94,22 +98,22 @@ We find repositories useful, but we acknowledge that they are not critical for y
 
```

Comment:
is the previous paragraph part of Jimmy's feedback or is this part of the book guidance @ardalis?

Reply:
That piece it outside of my eBook but the sentence starting with "We find repositories useful..." is not Jimmy's feedback, it's the book's recommendation.